src/helpers/pageInfo.js

```diff
@@ -30,12 +30,14 @@ const pageInfo = {
       url: '/u/verify-activities',
       component: VerifyActivities,
       menuIcon: VerifyActivitiesIcon,
+      allowedRoles: ['secretary', 'success', 'success ops'],
```

Comment:
I think we should have a format for our role names. the current naming convention seems too general and yet I think the roles should some sort of unique Identifiers
e.g. `SUCCESS_OPS`, `SUCCESS`, `SOCIETY_SEC`, `SOCIETY_PRES`
What do you think?